__tests__/test.js

```diff
@@ -75,6 +79,23 @@ describe('Hybrid-Cache', () => {
       Assert.strictEqual(cache2.get(Key), null)
     })
 
+    it('invalidate and update for every cache', async () => {
+      const Value2 = {
+        rich2: 'object2'
+      }
+      Assert.deepStrictEqual(cache.put(Key, Value), Value)
+      let cache2 = await Hybrid.Create(Topic, new MockPubSub(backingStore))
+      const p = new Promise( (resolve) => {
+        cache2.on('invalidate', (channel, key) => {
+          resolve(cache2.put(key, Value2))
```

Comment:
Add asserts for `channel` and `key`

Reply:
No need to `cache2.put` inside the `on`

Reply:
Add assert for `cache2.get(Key)`: should it be the value we `put` before we invalidate